CMakeLists.txt

```diff
@@ -4,12 +4,11 @@ cmake_minimum_required(VERSION 3.0)
 
 project(libtuntap)
 
-enable_testing()
+include(CTest)
```

Comment:
According to CMake documentation the variable `BUILD_TESTING` is set to `ON` by default.